src/collections/_documentation/platforms/python/troubleshooting.md

```diff
@@ -1,19 +1,58 @@
 ---
-title: Contextvars vs thread locals
+title: Troubleshooting
 sidebar_order: 11
 ---
 
-The Python SDK generally does its best to figure out how contextual data such
-as tags set with `sentry_sdk.set_tags` is supposed to flow along your control
-flow. In most usecases this is achieved perfectly fine using [thread
-locals](https://docs.python.org/3/library/threading.html#thread-local-data),
-but there are quite a few situations where this approach fails.
+This section contains various gotchas that are inherent to the Python SDK's
+design, do not necessarily affect the majority of users and do not really fit
+anywhere else.
 
-Read this document if you cannot figure out why context data is leaking across
-your HTTP requests, or when data is missing or popping up at the wrong place
-and time.
+The information here will help you answer these questions:
 
-## Python 2: thread locals and gevent
```

Comment:
Use Title case (though do not capitalize technical terms that aren't capitalized:) )